packages/docs/content/getting-started/for-developers.mdx

```diff
@@ -0,0 +1,385 @@
+
+### Using React components
+
+Inside a React project, you will import and use the components as you would any other React component. See the [Importing into a JavaScript project](#importing-into-a-javascript-project) below for more details.
+
+Outside a React project, your usage will depend on the framework you're using. [Our CDN example project](https://github.com/CMSgov/design-system/tree/main/examples/cdn) shows you how to use our React components in a plain HTML page. Note that for components that don't have interactive behaviors like [Badge](/components/badge), you may not need to invoke React at all. The best way to learn how to use our React components is to view [our example projects on GitHub](https://github.com/CMSgov/design-system/tree/main/examples/).
+
+More React prop documentation, examples, and guidance can be found on component documentation pages.
+
+#### Importing into a JavaScript project
+
+If you've installed the design system via npm, the components can be imported from the package entry point using the syntax below. **Note that the following example will display a different package name depending on the theme you have selected.**
+
+<ThemeContent onlyThemes={['core']}>
+
+```js
+import { Button, TextField } from '@cmsgov/design-system';
+```
+
+</ThemeContent>
+
+<ThemeContent onlyThemes={['healthcare']}>
+
+```js
+import { Button, TextField } from '@cmsgov/ds-healthcare-gov';
+```
+
+</ThemeContent>
+
+<ThemeContent onlyThemes={['medicare']}>
+
+```js
+import { Button, TextField } from 'ds-medicare-gov';
+```
+
+</ThemeContent>
+
+<Alert
+  heading="Performance note"
+  variation="warn"
+  className="ds-u-margin-bottom--2 ds-u-font-size--small"
+>
+  <p class="ds-c-alert__text">
+    This approach may result in a larger file size than intended. Depending on what module bundler
+    you use, all of the design system's React components may be included in the bundled file even if
+    you didn't specifically import them. This can be avoided by using{' '}
+    <a href="https://webpack.js.org/guides/tree-shaking/">tree shaking in Webpack</a>.
+  </p>
+</Alert>
+
+The `@cmsgov/design-system` package provides an ES modules version of our code to support Webpack 4's tree shaking optimizations. Our package is configured to use the `modules` property to let bundlers know to resolve imports from `@cmsgov/design-system` to the ES modules version when preferred. Read more about ESM in [Webpack's documentation](https://webpack.js.org/guides/author-libraries/#final-steps), or from [Rollup's proposal](https://github.com/rollup/rollup/wiki/pkg.module).
+
+Components can also be imported from their individual export file. This method carries no risk of including unused or extra code, as you are only importing from a specific file.
+
+```js
+import Button from '@cmsgov/design-system/dist/components/Button/Button';
+import TextField from '@cmsgov/design-system/dist/components/Button/TextField';
+```
```

Comment:
I'm also thinking about removing this whole section, @forrestbaer and @zarahzachz. I think that we want to encourage importing by name in most cases, especially if we want to be able to relocate things. I think most teams are using modern build tools, and many will be familiar with these kinds of JavaScript performance concerns.

Reply:
It might be more appropriate to bring it up in some kind of _performance for accessibility_ documentation in the future.

Reply:
For the purpose of succinctly getting people on the happy path, I vote we remove this.

Reply:
Yes, less is more here, vote to remove.